### PR TITLE
Add support for pasting the failed tests to Phabricator

### PR DESCRIPTION
**Pull request by kodchi**

### CODE REVIEW

File: barrybot.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def run_browser_tests( path, tag = None ):
     args = ['cd', path, '&&',
         'cd', 'tests/browser/', '&&',
         'bundle', 'exec', 'cucumber', 'features/', 
-        '--format', 'rerun'
+        # '--format', 'rerun'
```
Comment (jdlrobson) [inline barrybot.py]: need to remove

**Comment by phuedx** [comment]
@kodchi You could squash the two commits to make this PR somewhat cleaner.

**Comment by phuedx** [comment]
@kodchi Thanks for letting me know about `arc paste`!

**Comment by kodchi** [comment]
@phuedx, thanks I'll squash the patches.
@jdlrobson, OK, I'll post the URL only. It doesn't add anything extra.

**Comment by phuedx** [comment]
Should this be closed?

**Comment by jdlrobson** [comment]
Merged in https://github.com/jdlrobson/Barry-the-Browser-Test-Bot/commit/0879bd382ce21deabb5e813fc40a1d47f28cc3dd:)

**Pull request closed without merging**